react/components/AvailabilitySubscriber/Wrapper.js

```diff
@@ -0,0 +1,35 @@
+import React, { useContext } from 'react'
+import { ProductContext } from 'vtex.product-context'
```

Comment:
why not on `store-resources`?

Reply:
dunno, it was already like this. the app has been published already though, so 🤷‍♂

Reply:
I didn't know about `store-resources`, certainly it makes sense to move this context to there later